autoload/go/lsp/message.vim

```diff
@@ -19,7 +19,7 @@ function! go#lsp#message#Initialize(wd) abort
               \ },
               \ 'textDocument': {
                 \ 'hover': {
-                  \ 'contentFormat': ['plaintext'],
```

Comment:
I think this is still correct here. The "NoDocumentation" configuration should be set in the LSP config that `gopls` parses [here](https://github.com/golang/tools/blob/e713427fea3f98cb070e72a058c557a1a560cf22/internal/lsp/general.go#L230).

Reply:
Good call. Thanks for the extra set of eyes :bow: